docs/src/howto-guides/026_how_to_install_thin_edge_manually.md

```diff
@@ -106,10 +106,8 @@ Next, create the files and directories required by thin-edge.io and restart mosq
 
 ```shell
 sudo rc-service mosquitto stop
-sudo tedge --init
+sudo tedge init
```

Comment:
I would make the tedge user explicit here to give the user a hint that he can do differently.

```suggestion
sudo tedge init --user tedge --group tedge
```